physical_properties/pKa/microstates/oe_microstates.csv

```diff
@@ -0,0 +1,25 @@
+COc1ccc(cc1)c2c3c(=NCCO)[nH]c[nH+]c3oc2c4ccc(cc4)OC
```

Comment:
I'm having trouble figuring out what this file contains. What's `oe_microstates.txt`? Many of the files refer to a specific molecule SM01 through SM... , but `oe`?

Reply:
(There does not seem to be anything in the manifest anywhere which describes this; at least, I'm searching the change log for "oe_microstates" and the name of this file is the only place it occurs...)